hips/draw/simple.py

```diff
@@ -32,7 +34,7 @@ def draw_sky_image(geometry: WCSGeometry, tiles: Generator[HipsTile, Any, Any])
     image = np.zeros(geometry.shape)
     for tile in tiles:
         painter = SimpleTilePainter(geometry, tile)
-        image += painter.warp_image()
+        image += painter.warp_image(hips_survey.tile_width)
```

Comment:
Suggest to pass the `hips_survey` object to `SimpleTilePainter.__init__`, store it there as an attribute on self, and then access it where needed.